xml/System.Data.SqlClient/SqlDataReader.xml

```diff
@@ -656,9 +664,17 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Gets the read-only column schema collection.</summary>
```

Comment:
Here I'd copy the docs from [IDbColumnSchemaGenerator](https://docs.microsoft.com/en-us/dotnet/api/system.data.common.idbcolumnschemagenerator.getcolumnschema?view=netframework-4.8#System_Data_Common_IDbColumnSchemaGenerator_GetColumnSchema), at least for now.

```xml
<summary>Gets the column schema (<xref:System.Data.Common.DbColumn> collection).</summary>
<returns>The column schema (<xref:System.Data.Common.DbColumn> collection).</returns>
```

Reply:
This was modified as suggested by @mairaw above: https://github.com/dotnet/dotnet-api-docs/pull/3237#discussion_r328273510

Reply:
Ok, not critical either way for me, but I think that methods which implement am interface should in general repeat the docs from that interface unless there's a meaningful addition/diversion from that behavior (which doesn't seem to be the case here).

Reply:
I can revert to original text if @mairaw confirms?

Reply:
It's fine to revert if you prefer. We typically recommend having language neutral language (https://github.com/dotnet/dotnet-api-docs/wiki/Summary#guidelines), hence my suggestion.
But I'll approve either way since the text is already used.